docs/how_to/point_manipulator.md

```diff
@@ -8,28 +8,29 @@ Precisely positioning a point in 3D is a useful way to annotate specific positio
 
 ## Instructions
 
-1. Open the [points_manipulator.py](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/) example.
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/points_manipulator_plugin.py`. 
 
+2. Ensure you have the appropriate virtual environment active, change directories to the script location, and then you can execute the script using:
```

Comment:
Fair enough. Should we say "Activate your napari-threedee Python environment?" Or should we just not say anything about environment?

Reply:
I added that because it's what I had to do to run the script while testing all the How To's. Without that my system python gave an error.
Not sure how it can be avoided -- having a virtual env is from the napari install instructions.
One option would be to totally re-do the How To's to work like the render plane one with the virus sample data. No script is run, just menu options.